pom.xml

```diff
@@ -69,31 +66,23 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <!-- Spring Cloud dependencies-->
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-config</artifactId>
+        </dependency>
         <!-- <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
         </dependency> -->
-        <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>spring-cloud-starter-azure-spring-cloud-client</artifactId>
-            <version>2.1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-log4j2</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>applicationinsights-logging-log4j2</artifactId>
```

Comment:
Looks like you've removed the appinsights-log4j plugin - is this in the azure-spring-boot-bom now? Or don't we need it.

Reply:
No, I removed application insights since it caused some errors with SLF4J. There's a better way to introduce it but will save that for a separate PR. 
I forgot to remove the azure-bom but did it now. Thanks.